terraform/variables.tf

```diff
@@ -9,9 +9,15 @@ variable "aws_region" {
   default     = "us-east-1"
 }
 
-variable "bucket_name" {
-  description = "The name of the bucket that the user should be allowed to read from."
-  default     = "ncats-3rd-party-packages"
+variable "production_bucket_name" {
+  description = "The name of the S3 bucket that where the production Nessus package lives."
```

Comment:
Even though you are correct that there may be more than one version of the Nessus package in the bucket, I kept "package" as singular because from the perspective of this ansible role, it only cares about installing one package.  If it's pluralized, it makes it sound like the role is doing something with multiple packages.

I did however get rid of the extra "that" which wasn't supposed to be there- thanks for the sharp eyes!  See https://github.com/cisagov/ansible-role-nessus/pull/12/commits/c9febf4aa737ae50ee70bf4041538c745b901307.

Reply:
That's reasonable. Thanks for the explanation and stray word culling 👍